INTEGRATE.md

```diff
@@ -0,0 +1,33 @@
+# Local integration between Porta/System with Keycloak through Zync
+With these 3 components installed, you will be able to synchronize your applications in Porta with your Keycloak (Red Hat SSO)
+
+## Install Porta
+Follow [the installation guide](https://github.com/3scale/porta/blob/master/INSTALL.md).
+
+## Install Zync
+Follow [the installation guide](https://github.com/3scale/zync/blob/master/INSTALL.md).
+
+## Install Keycloak
+You can have a keycloak running locally with `docker run --name keycloak -d -p 8080:8080 -e KEYCLOAK_USER=admin -e KEYCLOAK_PASSWORD=p jboss/keycloak`
+So that docker container with keycloak will be accessible from your browser in `http://localhost:8080`.
```

Comment:
I do not know if there is a need to write this sentence. Docker user knows what that means. But you can say,

Note: Do not forget to map ports of your docker container with localhost `-p 8080:8080`

Reply:
@hallelujah Then I will simply remove the sentence, because I already said in the previous line the mapping of the ports... I don't need to say it twice 😄